build.gradle.kts

```diff
@@ -54,7 +50,27 @@ subprojects {
 
     repositories {
         jcenter()
-        google()
+        exclusiveContent {
+            forRepository {
+                google()
+            }
+            filter {
+                includeModuleByRegex("com\\.android.*", "(?!r8).*")
```

Comment:
can we use `excludeModule` for r8 to avoid regexp?

Reply:
yep, forgot about this one. thanks

Reply:
```
    ExclusiveContentRepository filter(Action<? super InclusiveRepositoryContentDescriptor> config);
```

only inclusive one available here, i understand the intent, but seems like non necessary limit